core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -225,14 +242,33 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     } yield ()
   }
 
-  def deleteBillingProject(projectName: RawlsBillingProjectName, ownerInfo: Map[String, String]): Future[PerRequestMessage] = {
+  def adminDeleteBillingProject(projectName: RawlsBillingProjectName, ownerInfo: Map[String, String]): Future[PerRequestMessage] = {
     // unregister then delete actual project in google
+    val ownerUserInfo = UserInfo(RawlsUserEmail(ownerInfo("newOwnerEmail")), OAuth2BearerToken(ownerInfo("newOwnerToken")), 3600, RawlsUserSubjectId("0"))
     for {
-      _ <- unregisterBillingProject(projectName, ownerInfo)
+      _ <- unregisterBillingProjectWithUserInfo(projectName, ownerUserInfo)
```

Comment:
unregisterBillingProjectWithOwnerInfo is used by AdminUnregisterBillingProject, and AdminUnregisterBillingProject do pass into the ownerInfo

Reply:
Ah, I see that now 👍. Is there a reason not to replace this call with `unregisterBillingProjectWithOwnerInfo` also?

Reply:
All the actual unregister logic goes to unregisterBillingProjectWithUserInfo(input is userInfo). 

When WithOwnerInfo will call unregisterBillingProjectWithUserInfo, WithOwnerInfo will convert map of ownerInfo into UserInfo. 
And for this method, it will just call unregisterBillingProjectWithUserInfo directly since we have UserInfo at hand. 

This do sounds confusion, any suggestion I can improve or any better name for those methods?  Thanks

Reply:
My vote would be to call `unregisterBillingProjectWithOwnerInfo` to reduce code duplication (lines 247 and 216), but it's such a small difference so really your call.